src/app/feed/feed-card/feed-card.component.html

```diff
@@ -26,6 +26,12 @@
 					[text]="itemText"
 					[config]="cardAutolinkerConfig" (click)="showLightBox.emit({show : 'hide'})"></feed-linker>
 			</div>
+			<div class="card-image" *ngFor="let image of feedItem.images let i = index">
```

Comment:
This looks good to me. Nice Work @Achint08!  Although as @Achint08 pointed out that URLs with pic.twitter.com are still appearing in the JSON, we can either filter those before using feedItem.images in the *ngFor. This can be merged for now and reproduced in a new issue.